src/ProductConstructionService/ProductConstructionService.Api/PcsStartup.cs

```diff
@@ -39,21 +39,26 @@
 using ProductConstructionService.DependencyFlow;
 using ProductConstructionService.ServiceDefaults;
 using Microsoft.Extensions.DependencyInjection.Extensions;
+using Octokit.Webhooks.AspNetCore;
+using Octokit.Webhooks;
+using ProductConstructionService.Api.Controllers;
 
 namespace ProductConstructionService.Api;
 
 internal static class PcsStartup
 {
     private const string SqlConnectionStringUserIdPlaceholder = "USER_ID_PLACEHOLDER";
+    private const string GitHubWebHooksPath = "/api/webhooks/incoming/github";
 
-    private static class ConfigurationKeys
+    internal static class ConfigurationKeys
```

Comment:
This is what I was trying to avoid (details in the other thread)

Reply:
fixed